.github/workflows/tests/justfile_tests.bats

```diff
@@ -29,3 +29,30 @@ setup() {
     [ "$status" -eq 0 ]
     [ ! -f "${HOME}/.config/no-show-user-motd" ]
 }
+
+@test "Ensure bash lockdown works" {
+    BASH_ENV_FILES=(
+        "$HOME/.bashrc"
+        "$HOME/.bash_profile"
+        "$HOME/.config/bash-completion"
+        "$HOME/.profile"
+        "$HOME/.bash_logout"
+        "$HOME/.bash_login"
+      )
+    BASH_ENV_DIRS=(
+        "$HOME/.bashrc.d/"
+        "$HOME/.config/environment.d/"
+      )
+    for file in "${BASH_ENV_FILES[@]}"; do
+        before=$(lsattr "$file" 2>/dev/null)
```

Comment:
1. `toggle-bash-environment-lockdown` affects all files at once, not one by one. `before` and `after` must capture the status of all files, and there must be only one call to `toggle-bash-environment-lockdown`.
2. Directories down below are tested specifically for immutability. For some reason all of files' attributes are compared.